products/dotnet.md

```diff
@@ -32,8 +32,9 @@ releases:
 -   releaseCycle: "Core 3.1"
     cycleShortHand: "3.1"
     lts: true
-    latest: "3.1.25"
-    eol: 2022-12-03
+    latest: "3.1.26"
```

Comment:
This ought to be getting updated automatically, unsure why this isn't.

Probably these releases: https://github.com/dotnet/core/releases/tag/v3.1.200, https://github.com/dotnet/core/releases/tag/v3.1.201

Will check

Reply:
So the `3.1.abc` releases seem to be tagged only for SDK purposes.